src/components/general/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import Accordion from './index';
+import withFusionStory from '../../../../.storybook/withFusionStory';
+
+const AccordionStory = () => {
+    const [isOpenFirst, setIsOpenFirst] = React.useState(false);
+    const [isOpenSecond, setIsOpenSecond] = React.useState(false);
+
+    const accordionContent = (
+        <div>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin vehicula condimentum est.
+            Sed porttitor metus nec aliquet fermentum. Sed ut nibh maximus, rutrum nibh id, posuere
+            ligula. Morbi a pulvinar ligula, ut imperdiet ipsum. Mauris dui risus, consequat eget
+            nibh sed, bibendum tempor diam. Aliquam sed elit tempor, pretium velit sit amet,
+            faucibus risus. Aenean maximus ipsum eros, eget facilisis enim rhoncus vel. Sed nec
+            venenatis nisi. Morbi feugiat auctor neque. Proin mauris mauris, maximus ut porta eget,
+            rhoncus ut dui. Pellentesque tempus metus non lacus hendrerit, quis tristique libero
+            feugiat. Quisque feugiat ultricies tempor. Cras vel diam gravida, laoreet mi facilisis,
+            blandit tortor.
+        </div>
+    );
+    return (
+        <div style={{ width: 500 }}>
+            <Accordion
```

Comment:
Could we change this to be an Accordion component that contains all of the collapsible items? Like `<Accordion><AccordionItem isOpen={isOpenFirst} onChange={...} label="Accordion header">{accordionContent}</AccordionItem><AccordionItem isOpen={isOpenSecond} onChange={isOpenSecond} label="Right action" actionDirection="right">{accordionContent}</AccordionItem></Accordion>` 🤔

Reply:
But would that main Accordion component contain any specific logic/styling or would it just return its children?

Reply:
It would contain styling (how the items relate to each other) and _could_ contain logic if we need something like only one item open at the time

Reply:
I see I see